docs/upload-your-snap.md

```diff
@@ -1,69 +1,154 @@
-# Upload to the store
+---
+title: "Release your snap"
+---
 
-So you've been working hard on your snap, and you finally have it to the point
-where you're ready to share it with the world? Great! You can use Snapcraft to
-upload it directly to the store.
 
-## The grade option
+In order to share your snaps with the world, you need to publish them in a
+store. This page describes the publication process you will follow for the
+Ubuntu-hosted Store.
 
-`grade` in your `snapcraft.yaml` file can be either `stable` or `devel`. It
-essentially tags the resulting snap for life with the grade it should have.
+Although, snaps are not tied to a specific type of store and you can host
+them any way you want, see the [Store](/docs/core/store) page for more
+details.
```

Comment:
Analytics on snapcraft.io show they don't get lost and this gives the context they need to understand store accounts and have an overivew of how the store works. Although, the link should be changed to a link to snapcraft.io (http://snapcraft.io/docs/core/store), since /docs/core/store won't work here.